scripts/partition_manager/partition_manager.rst

```diff
@@ -49,6 +49,21 @@ This file must be stored in the same folder as the main :file:`CMakeLists.txt` f
    The root application does not need to define a :file:`pm.yml` file, because its partition size and placement is implied by the size and placement of the sub-image partitions.
    If a root application defines a :file:`pm.yml` file, it is silently ignored.
 
+A subsystem can also provide Partition Manager configuration.
+This is done by adding the path to the :file:`pm.yml` file to the global property ``PM_SUBSYS``.
+Subsystem Partition Manager configurations can not define image partitions.
+The listing below shows how to properly define the Partition Manager configuration for a subsystem.
+This could would be placed in the :file:`CMakeLists.txt` of that subsystem.
+
```

Comment:
Mention that duplicate definitions do not create multiple regions, and that they must be identical.